Explorer/Assets/DCL/Infrastructure/SceneRuntime/Apis/Modules/CommsApi/CommsApiWrap.cs

```diff
@@ -0,0 +1,62 @@
+using Cysharp.Threading.Tasks;
+using DCL.Multiplayer.Connections.RoomHubs;
+using JetBrains.Annotations;
+using LiveKit.Proto;
+using SceneRunner.Scene.ExceptionsHandling;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SceneRuntime.Apis.Modules.CommsApi
+{
+    public class CommsApiWrap : IJsApiWrapper
```

Comment:
Please implement the IV8HostObject interface.

Reply:
Please change the code so that it does not allocate memory. Instead of creating a GetActiveVideoStreamsResponse, assemble the json string directly using a StringBuilder or a JsonWriter. Or even better: change the JavaScript API to take a buffer instead of a string, write binary data directly to that, and then use various parsing and decoding functions on the JavaScript side to construct the JavaScript object.

Reply:
If you need help with any of that, don't hesitate to ask.

Reply:
Alternatively, you can argue that this code runs rarely enough that performance is not a concern.